dmscripts/helpers/supplier_data_helpers.py

```diff
@@ -329,17 +326,25 @@ def get_supplier_ids_from_args(args):
     return supplier_ids
 
 
-@lru_cache()
```

Comment:
Why are we removing the cache?

Reply:
Ah, good spot 👀 
I removed this as I assumed it was something to do with caching the HTTP response but now I see it is a more generic thing, I have restored it now in 7e05fc0